check-trailing-whitespaces

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-files_with_trailing_spaces=$(find . -type f -exec grep -EIHn "\\s$" {} \; | sort -fh)
+files_with_trailing_spaces=$(find . -type f -not -path "./.git/*" -not -path "./.svn/*" -exec grep -EIHn "\\s$" {} \; | sort -fh)
```

Comment:
IMO it's better to ignore those directories even if they are in a deeper level than the current one, especially for SVN as it creates a `.svn` in _all_ directories of a working copy (until version 1.7).

Reply:
ok, so mumbo-jumbo needed here is out of the my limit :(